### PR TITLE
Add Mercedes-Benz as known instance to innersource-portal.md

### PR DESCRIPTION
**Pull request by spier**
This was written about publically here:
https://opensource.mercedes-benz.com/news/sponsor_innersource_commonsoss

@wgehring is there anything that you would like to add about your implementation of the InnerSource Portal? Also could you review this PR so that we know if it is good to be published in our patterns book?

### CODE REVIEW

**Comment by wgehring** [comment]
Yes that works! Thanks very much for including us. 
Nothing to add for now - if we have some more experience, then I can perhaps add this later.

File: patterns/2-structured/innersource-portal.md [hunk 1]
```diff
@@ -76,8 +76,9 @@ A [reference implementation](https://github.com/SAP/project-portal-for-innersour
 ![Santander InnerSource Portal](../../assets/img/santander_portal.png "Banco Santander InnerSource Portal")
 
 * **Airbus** used the [SAP Portal](https://github.com/SAP/project-portal-for-innersource) as a Proof of Concept. It is now using the [Bazaar plugin](https://github.com/backstage/backstage/blob/master/plugins/bazaar/README.md) of [Backstage](https://backstage.io) as the latter became the official developer experience tool internally. It provides a convenient self-registering capability for all the divisions.
-
 * **Mercado Libre** use an instance of the [SAP portal](https://github.com/SAP/project-portal-for-innersource) to discover existing InnerSource projects within the organization.
+* **Mercedes-Benz** is [using](https://opensource.mercedes-benz.com/news/sponsor_innersource_commonsoss) the SAP reference implementation mentioned above as their InnerSource Portal
+
```
Comment (spier) [inline patterns/2-structured/innersource-portal.md]:
```suggestion
* **Mercedes-Benz** is [using](https://opensource.mercedes-benz.com/news/sponsor_innersource_commonsoss) the SAP reference implementation mentioned above for their InnerSource Portal.
```

**Comment by spier** [comment]
@wgehring awesome. Thanks for the quick confirmation. 

This is now live here:
https://patterns.innersourcecommons.org/p/innersource-portal

Btw do you want us to refer to you in the pattern as "Mercedes-Benz Tech Innovation", as it says on the sponsors page m, or does "Mercedes-Benz" do? Personally I think the latter is more straightforward as that is the brand that people know but of course we will adapt to your preference.

Also is there somebody that I could interview to explore if there are other patterns that you use, where we could add you as known instances?

**Comment by wgehring** [comment]
Mercedes-Benz is good! We are actually using it for the entire big company. :-)

You could interview me if you like. :-)

**Pull request merged**